modules/core/arrow-extras-data/src/test/kotlin/arrow/data/FingerTreeTest.kt

```diff
@@ -16,6 +16,7 @@ import io.kotlintest.properties.Gen
 import io.kotlintest.properties.forAll
 import io.kotlintest.shouldBe
 import io.kotlintest.specs.StringSpec
+import java.util.*
```

Comment:
@pakoito is it ok to have references to java? MPP?

Reply:
For the tests it's okay, as they're JVM-only because of the runner.

Reply:
Okay, thanks! 👍